Command/GeneratorCommand.php

```diff
@@ -55,8 +55,13 @@ protected function getSkeletonDirs(BundleInterface $bundle = null)
             $skeletonDirs[] = $dir;
         }
 
-        $skeletonDirs[] = __DIR__.'/../Resources/skeleton';
-        $skeletonDirs[] = __DIR__.'/../Resources';
+        $bundleDirs = $this->getContainer()->get('kernel')
+                           ->locateResource('@SensioGeneratorBundle/Resources/skeleton', null, false);
+
+        foreach ($bundleDirs as $singleBundleDir) {
+            $skeletonDirs[] = $singleBundleDir;
+            $skeletonDirs[] = dirname($singleBundleDir);
+        }
```

Comment:
Hi @fabpot , will the [#235 PR](https://github.com/sensiolabs/SensioGeneratorBundle/pull/235) be merged? It's useful. Thanks!